pkg/oc/cli/describe/describer.go

```diff
@@ -1879,6 +1879,13 @@ func describeSecurityContextConstraints(scc *securityapi.SecurityContextConstrai
 		fmt.Fprintf(out, "  Allowed Seccomp Profiles:\t%s\n", stringOrNone(strings.Join(scc.SeccompProfiles, ",")))
 		fmt.Fprintf(out, "  Allowed Volume Types:\t%s\n", fsTypeToString(scc.Volumes))
 		fmt.Fprintf(out, "  Allowed Flexvolumes:\t%s\n", flexVolumesToString(scc.AllowedFlexVolumes))
+
+		if len(scc.AllowedUnsafeSysctls) > 0 {
+			fmt.Fprintf(out, "  Allowed Unsafe Sysctls:\t%s\n", sysctlsToString(scc.AllowedUnsafeSysctls))
+		}
+		if len(scc.ForbiddenSysctls) > 0 {
+			fmt.Fprintf(out, "  Forbidden Sysctls:\t%s\n", sysctlsToString(scc.ForbiddenSysctls))
+		}
```

Comment:
Looks like we always show the fields. If their content is empty, we just represent it as `<all>` or `<none>`. So, could you remove these `if`-s then?

Reply:
Fixed :)